docs/migrating-v1alpha1.Run-to-v1beta1.CustomRun.md

```diff
@@ -0,0 +1,87 @@
+<!--
+---
+linkTitle: "Migrating from v1alpha1.Run to v1beta1.CustomRun"
+weight: 4000
+---
+-->
+
+# Migrating from v1alpha1.Run to v1beta1.CustomRun
+
+This document describes the differences between `v1alpha1.Run` and `v1beta1.CustomRun`.
+
+## Changes to fields
+
+Comparing `v1alpha1.Run` with `v1beta1.CustomRun`, the following fields have been changed:
+
+| Old field              | New field         |
+| ---------------------- | ------------------|
+| `spec.spec`            | [`spec.customSpec`](#changes-to-the-specification) |
+| `spec.ref`             | [`spec.customRef`](#changes-to-the-reference) |
+| `spec.podTemplate`     | removed |
```

Comment:
please add section showing that users can support `podtemplate` in their own custom task spec, and reference that section in this table

for example:

```yaml
apiVersion: tekton.dev/v1beta1
kind: CustomRun
metadata:
  name: customrun-with-podtemplate
spec:
  customSpec:
    apiVersion: example.dev/v1beta1
    kind: Example
    spec:
      podTemplate: 
        securityContext:
        runAsUser: 1001


```

Reply:
SG, added, thanks!